mesheryctl/internal/cli/root/system/token.go

```diff
@@ -30,7 +30,12 @@ var tokenCmd = &cobra.Command{
 var addTokenCmd = &cobra.Command{
 	Use:   "add",
 	Short: "Add a token to config.yaml",
-	Args:  cobra.ExactArgs(1),
+	Long: `
```

Comment:
```suggestion
	Example: `
```
"Long" is the longer version of the description.

Reply:
okay,will do the change

Reply:
@piyushsingariya  did the change